docs/install.rst

```diff
@@ -5,58 +5,112 @@ Installation
 
-$ conda create --name plenoptic pip python=3.9
-$ conda activate plenoptic
-$ pip install plenoptic
+If you are unfamiliar with python environment management, we recommend :ref:`conda`.
+
+Our dependencies include `pytorch <https://pytorch.org/>`_ and `pyrtools <https://pyrtools.readthedocs.io/en/latest/>`_. Installation should take care of them (along with our other dependencies) automatically, but if you have an installation problem (especially on a non-Linux operating system), it is likely that the problem lies with one of those packages. `Open an issue <https://github.com/LabForComputationalVision/plenoptic/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=>`_ and we'll try to help you figure out the problem!
+
+.. _conda:
 
-Our dependencies include `pytorch <https://pytorch.org/>`_ and `pyrtools <https://pyrtools.readthedocs.io/en/latest/>`_. Installation should take care of them (along with our other dependencies) automatically, but if you have an installation problem (especially on a non-Linux operating system), it is likely that the problem lies with one of those packages. `Open an issue <https://github.com/LabForComputationalVision/plenoptic/issues>`_ and we'll try to help you figure out the problem!
+Installing with conda
+---------------------
+
+If you wish to follow these instructions and do not have ``conda`` installed on your machine, I recommend starting with `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_.
 
-You can also install it directly from source to have a local editable copy. This is most useful for developing (for more info, see `our contributing guide <https://github.com/LabForComputationalVision/plenoptic/blob/main/CONTRIBUTING.md>`_) or if you want to use the most cutting-edge version::
+Once you have ``conda`` correctly installed and on your path, run the following to create a new virtual environment and install plenoptic::
 
-$ conda create --name plenoptic pip python=3.9
+$ conda create --name plenoptic pip python=3.11
 $ conda activate plenoptic
-$ # clone the repository
-$ git clone https://github.com/LabForComputationalVision/plenoptic.git
-$ cd plenoptic
-$ # install in editable mode with `-e` or, equivalently, `--editable`
-$ pip install -e .
+$ conda install plenoptic -c conda-forge
+
+.. _pip:
+
+Installing with pip
+-------------------
+
+Wile ``conda`` handles both virtual environment management **and** package installation, ``pip`` only installs packages; you'll need to use some other system to manage your virtual environment. You can use ``conda`` to manage your virtual environment and ``pip`` to install packages, but note that when using pip with a conda environment, all pip commands should come after all the conda commands (e.g., you shouldn't run ``conda install matplotlib`` after ``pip install plenoptic``), because conda is unaware of any changes that pip makes. See `here <https://www.anaconda.com/blog/using-pip-in-a-conda-environment>`_ for more details.
```

Comment:
```suggestion
While ``conda`` handles both virtual environment management **and** package installation, ``pip`` only installs packages; you'll need to use some other system to manage your virtual environment. You can use ``conda`` to manage your virtual environment and ``pip`` to install packages, but note that when using pip with a conda environment, all pip commands should come after all the conda commands (e.g., you shouldn't run ``conda install matplotlib`` after ``pip install plenoptic``), because conda is unaware of any changes that pip makes. See `here <https://www.anaconda.com/blog/using-pip-in-a-conda-environment>`_ for more details.
```